src/services/activityReports.js

```diff
@@ -1043,6 +1042,21 @@ export async function setStatus(report, status) {
  * @returns {*} Grants and Other entities
  */
 export async function possibleRecipients(regionId, activityReportId = null) {
+  // If we have an activityReportId get the startDate of the report.
+  // We have to consider that we have a report with no ActivityRecipients (ie no way to join).
+  const reportWithStartDate = activityReportId ? await ActivityReport.findOne({
+    attributes: ['startDate'],
+    where: {
+      id: activityReportId,
+    },
+  }) : null;
+
+  // If we have a valid startDate subtract 61 days from it.
+  let startDate = null;
+  if (reportWithStartDate && reportWithStartDate.startDate) {
+    startDate = moment(reportWithStartDate.startDate).subtract(61, 'days').format('YYYY-MM-DD');
```

Comment:
I don't think this is correct business logic. If we have the actual startDate for the AR, then we just want to get the grants active on that date. It's when we do _not_ have a startDate that we hunt back 61 days from the day the AR record is opened

Reply:
This code has been removed. We just added a select in the case. See latest updtes.